### PR TITLE
Update the expected hammer commands and options

### LINKED ISSUE #2150: Automation failure : .test_hammer_options
this test is failing for snap2 GA
Test Names:
test_hammer.HammerCommandsTestCase.test_hammer_options

This test is failing for some assertion error:

```
$  nosetests -c robottelo.properties -m test_hammer_options /home/tkolhar/git/robottelo/tests/foreman/cli/test_hammer.py
self.assertIn(option, info['options'])
AssertionError: {
    u'help': u'Set output format. One of [yaml, base, table, csv, silent, json]',
    u'name': u'output',
    u'shortname': None,
    u'value': u'ADAPTER',
} not found in [
    {u'shortname': None, u'help': u'Get list of possible endings', u'value': u'LINE', u'name': u'autocomplete'},
    {u'shortname': None, u'help': u'Output as CSV (same as --output=csv)', u'value': None, u'name': u'csv'},
    {u'shortname': None, u'help': u'Character to separate the values', u'value': u'SEPARATOR', u'name': u'csv-separator'},
    {u'shortname': None, u'help': u'Explicitly turn interactive mode on/off One of true/false, yes/no, 1/0.', u'value': u'INTERACTIVE', u'name': u'interactive'},
    {u'shortname': None, u'help': u'Set output format. One of [base, table, silent, csv, yaml, json]', u'value': u'ADAPTER', u'name': u'output'},
    {u'shortname': None, u'help': u'Show ids of associated resources', u'value': None, u'name': u'show-ids'},
    {u'shortname': None, u'help': u'show version', u'value': None, u'name': u'version'},
    {u'shortname': u'c', u'help': u'path to custom config file', u'value': u'CFG_FILE', u'name': u'config'},
    {u'shortname': u'd', u'help': u'show debugging output', u'value': None, u'name': u'debug'},
    {u'shortname': u'h', u'help': u'print help', u'value': None, u'name': u'help'},
    {u'shortname': u'p', u'help': u'password to access the remote system', u'value': u'PASSWORD', u'name': u'password'},
    {u'shortname': u'r', u'help': u'force reload of Apipie cache', u'value': None, u'name': u'reload-cache'},
    {u'shortname': u's', u'help': u'remote system address', u'value': u'SERVER', u'name': u'server'},
    {u'shortname': u'u', u'help': u'username to access the remote system', u'value': u'USERNAME', u'name': u'username'},
    {u'shortname': u'v', u'help': u'be verbose', u'value': None, u'name': u'verbose'}
]
```

please fix it 
thanks

### PR DESCRIPTION
**Pull request by elyezer**
Update the expected hammer commands and options to match the current
status.

Also check for some known bugs and deal with the related commands
accordingly.

Closes #2150

### CODE REVIEW

**Comment by elyezer** [comment]
Will get fresh results when the new build is installed.

File: tests/foreman/cli/test_hammer.py [hunk 1]
```diff
@@ -39,6 +39,10 @@ def _traverse_command_tree(self, command):
         the expected options are present.
 
         """
+        if (command == 'hammer content-host errata list' and
+                bz_bug_is_open(1219593)):
+            # Skip until the bug gets fixed
+            return
```
Comment (sthirugn) [inline tests/foreman/cli/test_hammer.py]: :+1:

**Comment by sthirugn** [comment]
APT

**Comment by elyezer** [comment]
Got the results on the latest build:

```
py.test tests/foreman/cli/test_hammer.py
=================================== test session starts ===================================
platform darwin -- Python 2.7.6 -- py-1.4.26 -- pytest-2.6.4
collected 1 items

tests/foreman/cli/test_hammer.py .

=============================== 1 passed in 1247.67 seconds ===============================
```

**Comment by elyezer** [comment]
Rebased

**Comment by Ichimonji10** [comment]
APT

**Pull request merged**